pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "foursight-core"
-version = "4.2.0"
+version = "4.3.0.1b1"  # TODO: To become 4.3.0
```

Please take this suggestion only if you didn't already tag the beta, since you don't want to move backward in time, but I'll make this as if we're in time:
```suggestion
version = "4.2.0.1b1"  # TODO: To become 4.3.0
```

In our local rules, the beta version should be epsilon higher than what it's building from (not targeting) and strictly less than what it's planning to release to. The whole point is to work around some bugs some tools have with betas where they don't realize they come _before_ the numeric part, so if the numeric part is 4.3.0.1, you are unintentionally assuring the bug our numbering is trying to work around. :)